php-frontend/src/test/java/org/sonar/plugins/php/api/tests/PhpCheckTestUtils.java

```diff
@@ -39,20 +39,7 @@
 import java.util.Map;
 
 /**
- * Base helper class for checks unit test.
- * <p/>
- * Usage example:
- * <p/>
- * Check unit test class
- * <pre>
- *   public class MyCustomCheck extends PHPCheckTest {
- *
- *     {@literal@}Test
- *     public void test() {
- *       check(new MyCustomCheck(), new File("code_sample.php"));
- *     }
- *   }
- * </pre>
```

Comment:
@vilchik-elena If my memory is correct, the doc was saying that the check class should inherit from this class. Before I made the changes, this was already no longer true.